### ISSUE TITLE
Feature or API endpoint for automating updates of multiple datasets

### ISSUE THREAD

**Issue by pragermh** [gbif/ipt#2088]
This issue is related to [#1973](https://github.com/gbif/ipt/issues/1973) and [#1977](https://github.com/gbif/ipt/issues/1977), i.e. concerns automation of update tasks in the IPT. In my particular case, I update the taxonomic annotation of multiple DNA derived datasets in our source database, and then have to manually republish each of these in the IPT. At the moment, we have very few of these datasets so this is not a problem, but in the future it would be great if there was a way to automate this task. I understand that I can already set a regular auto-publishing interval in the IPT, but my annotation updates will follow a release of a new version of the taxonomic reference databases that I use, and may not be that regular, so an on-demand feature would be more useful. So, specifically, what I would like to suggest is a feature or an API endpoint where one can specify a list of datasets that should be republished with a common change summary.

**Comment by mike-podolskiy90** [comment]
@pragermh Thank you for contacting us.
What do you think if we improve auto-publication with a flag "Only publish if data/metadata was changed"? It'd be quite easy to implement. 
We can also consider adding a new API endpoint or something else.

**Comment by pragermh** [comment]
Thanks @mike-podolskiy90, that sounds useful. Related question: I experimented a little with a modified version of @fujiokae :s python script against GBIF:s demo IPT (https://ipt.gbif.org/) because we don't have a Swedish test IPT. Is that OK, you think? I only managed to change the Last modified date of one of my datasets without actually re-publishing anything, though.

**Comment by mike-podolskiy90** [comment]
Yes, of course, you can test there. There is also ipt.gbif-uat.org where we run the latest development version

**Comment by mike-podolskiy90** [comment]
Will be implemented here #2730
